bitcoin/src/address/mod.rs

```diff
@@ -430,7 +430,7 @@ impl Address {
     ///
     /// This is a segwit address type that looks familiar (as p2sh) to legacy clients.
     pub fn p2shwpkh(pk: CompressedPublicKey, network: impl Into<NetworkKind>) -> Address {
-        let builder = script::Builder::new().push_int(0).push_slice(pk.wpubkey_hash());
+        let builder = script::Builder::new().push_int(0).unwrap().push_slice(pk.wpubkey_hash());
```

Comment:
We could use `push_uint` here and not use `unwrap`

Reply:
This file is out of blockdata so cannot use `push_uint`